addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentsList.js

```diff
@@ -143,11 +143,10 @@ class ScEnvironmentsList extends React.Component {
     );
   }
 
-  renderMain(appStreamProjectIds) {
+  renderMain() {
     const store = this.envsStore;
     const selectedFilter = this.selectedFilter;
-    let list = store.filtered(selectedFilter);
-    list = this.isAppStreamEnabled ? _.filter(list, env => _.includes(appStreamProjectIds, env.projectId)) : list;
+    const list = store.filtered(selectedFilter);
```

Comment:
Why don't we need to filter the env based on `isAppStreamEnabled` anymore?

Reply:
Because we want to give users an option to terminate the non-AppStream environments.